packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1114,7 +1124,11 @@ class ListView extends BoxScrollView {
     ScrollViewKeyboardDismissBehavior keyboardDismissBehavior = ScrollViewKeyboardDismissBehavior.manual,
     String? restorationId,
     Clip clipBehavior = Clip.hardEdge,
-  }) : childrenDelegate = SliverChildListDelegate(
+  }) : assert(
+         !(itemExtent != null && prototypeItem != null),
+         'You can only pass [itemExtent] or [prototypeItem], not both',
```

Comment:
Avoid the square brackets in the error messages, since it's not markdown. Also, end the message with a period.

Reply:
(below also)

Reply:
This is like the previous one. Refer to `PopupMenuButton` for a sample.

Maybe it should be documented in the Flutter Style Guide? I think people tend to rely on context when the rule is not specified on the guide, like I did...